stdlib/LibGit2_jll/Project.toml

```diff
@@ -9,7 +9,7 @@ Libdl = "8f399da3-3557-5675-b5ff-fb832c97cbdb"
 Artifacts = "56f22d72-fd6d-98f1-02f0-08ddc0907c33"
 
 [compat]
-julia = "1.8"
+julia = "1.9"
```

Comment:
I wasn't entirely sure about that, but it seemed like it needed to match the change you made in the other PR.